incubator/drone/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 name: drone
 home: https://drone.io/
-version: 0.1.0
-appVersion: 0.8.1
-description: Drone CI
+icon: http://docs.drone.io/logo.svg
+version: 0.1.1
```

Comment:
This changes the API to the chart. For example, the parameter went from `image.repository` to `image.server.repository`. Since a 1.0 has not come out yet the minor version should be bumped rather than the patch.

Reply:
You are completely right